apis/trips/tripsgo/versionService.go

```diff
@@ -0,0 +1,15 @@
+package tripsgo
+
+import (
+	"net/http"
+	"fmt"
+	"os"
+)
+
+func versionGet(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/plain; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
+
+	version := os.Getenv("APP_VERSION")
+	fmt.Fprintf(w, version)
+}
```

Comment:
Linter, requires New Line at the end of file

Reply:
No problem, added the newline.